website/api/controllers/entrance/signup.js

```diff
@@ -205,17 +216,6 @@ the account verification message.)`,
       sails.log.warn(`When a new user signed up, a lead/contact could not be verified in the CRM for this email address: ${newEmailAddress}. Raw error: ${err}`);
       return;
     });
-    // If billing feaures are enabled, save a new customer entry in the Stripe API.
-    // Then persist the Stripe customer id in the database.
-    if (sails.config.custom.enableBillingFeatures) {
```

Comment:
We decided to keep a version of this `if` check because it's still used elsewhere.  If not configured throw